themes/default/css/wide.css

```diff
@@ -1182,7 +1182,7 @@ table.representationAnnotationListItem {
 }
 
 #objectRepresentationOverlay div.close {
-	background:url(../graphics/buttons/close.png) no-repeat; 
+	background:url(../graphics/icons/glyphicons_192_circle_remove.png) no-repeat;
```

Comment:
We should replace these icons with Font-Awesome icons (using caNavIcon()?)

Reply:
I will remove this commit, it shouldn't be in this PR.